src/diagonal.works/b6/cmd/b6/js/static/b6.css

```diff
@@ -458,6 +458,67 @@ li div.collection-feature {
     align-items: center;
 }
 
+.block .block-container.histogram {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.block .block-container.histogram .range-icon {
+    flex: 0.1 0 auto;
+    background-image: url("/images/histogram-range.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 70%;
+    margin-right: 5px;
+}
+
+.block .block-container.histogram .index-0 {
+    filter: invert(86%) sepia(14%) saturate(359%) hue-rotate(290deg) brightness(102%) contrast(91%);
```

Comment:
I imagine you enjoyed learning about that trick. Is there a reason we're using images for the swatch and bar, rather than just a styled div with absolute size and a background colour?

Reply:
fml

done